src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -4,75 +4,87 @@
 import * as React from 'react';
 import {Dimensions, FlatList, KeyboardAvoidingView, Platform, StyleSheet, View} from 'react-native';
 
-import AnonymousChatHeader from '../../components/AnonymousChat/AnonymousChatHeader';
 import AnonymousInputMessage from '../../components/Chat/AnonymousInputMessage';
 import BaseChatItem from '../../components/AnonymousChat/BaseChatItem';
-import useAnonymousChatScreenHook from '../../hooks/screen/useAnonymousChatScreenHook';
+import ChatDetailHeader from '../../components/AnonymousChat/ChatDetailHeader';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been updated to reflect the new components and hooks being used. This is a good change as it ensures that the correct components and hooks are being utilized in this file.

```diff
- import AnonymousChatHeader from '../../components/AnonymousChat/AnonymousChatHeader';
+ import ChatDetailHeader from '../../components/AnonymousChat/ChatDetailHeader';
- import useAnonymousChatScreenHook from '../../hooks/screen/useAnonymousChatScreenHook';
+ import useChatScreenHook from '../../hooks/screen/useChatScreenHook';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->